deploy/charts/rook-ceph-cluster/templates/ingress.yaml

```diff
@@ -31,6 +31,9 @@ spec:
               {{- else }}
               servicePort: http-dashboard
               {{- end }}
+  {{- if .Values.ingress.dashboard.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.dashboard.host.path | default "nginx" }}
```

Comment:
No need for the default if this will only be evaluated when ingressClassName is set, right?
```suggestion
  ingressClassName: {{ .Values.ingress.dashboard.host.path }}
```

Reply:
Right, makes sense.

Reply:
Actually, shouldn't the value be set like this?
```suggestion
  ingressClassName: {{ .Values.ingress.dashboard.ingressClassName }}
```

Reply:
forgot to update after copy paste.